supersid/find_alsa_devices.py

```diff
@@ -517,17 +518,19 @@ def __init__(self, verbose):
             self.E_INVALID_LENGTH = 5
             self.E_INVALID_DATA_LENGTH = 6
             self.E_RECORDED_ALL_ZEROS = 7
-            self.F_NOT_IMPLEMENTED = 8
+            self.E_MONO = 8
+            self.F_NOT_IMPLEMENTED = 9
             self.verbose = verbose
 
         def test_configuration(
                 self,
                 pcm_device,
                 rate,
                 format,
-                periodsize):
+                periodsize,
+                channels):
             try:
-                channels = 1
+                assert(channels in [1, 2])
```

Comment:
Assert should not have parens. https://stackoverflow.com/questions/3112171/python-assert-with-and-without-parenthesis

Reply:
ok, I'll refactor all asserts in the project